unlock-js/src/__tests__/v01/getLock.test.js

```diff
@@ -81,6 +81,12 @@ describe('v01', () => {
         )
       )
 
+      nock.ethCallAndYield(
+        contractMethods.name.encode([]),
+        checksumLockAddress,
+        resultEncoder.encode(['string'], [utils.toRpcResultString('My Lock')])
```

Comment:
it would be good to extract `'My Lock'` into a variable to use below as well

Reply:
dng I missed that comment sorry :/ I 'll do it in a later PR.